src/libslic3r/GCodeWriter.cpp

```diff
@@ -224,6 +224,8 @@ std::string GCodeWriter::set_pressure_advance(double pa) const
     else{
         if (FLAVOR_IS(gcfKlipper))
             gcode << "SET_PRESSURE_ADVANCE ADVANCE=" << std::setprecision(4) << pa << "; Override pressure advance value\n";
+        else if(FLAVOR_IS(gcfRepRapFirmware))
```

Comment:
👍

Reply:
I cant find the variable for Extruder ID.  For M572 D0 >- extruder ID

Reply:
Multi-extruder support is another rabbit hole ;)